manager/volume.go

```diff
@@ -603,6 +603,30 @@ func (m *VolumeManager) CancelExpansion(volumeName string) (v *longhorn.Volume,
 	return v, nil
 }
 
+func (m *VolumeManager) OfflineRebuild(volumeName string, rebuildState longhorn.VolumeOfflineRebuild) (v *longhorn.Volume, err error) {
```

Comment:
The naming of parameter `rebuildState` is a bit weird, but I don't have a better suggestion.

```suggestion
func (m *VolumeManager) UpdateOfflineRebuild(volumeName string, rebuildState longhorn.VolumeOfflineRebuild) (v *longhorn.Volume, err error) {
```

Reply:
Renamed the API name and modified the name `rebuildState` to `offlineRebuildMode`, WDYT?
Thanks.